.gitignore

```diff
@@ -0,0 +1,4 @@
+*~
+
+# Mac files
+.DS_Store
```

Comment:
Works for me! I used to add a bunch of stuff like this to repos, but then (IIRC) Steven pointed out that we should probably put this stuff in our `~/.config/git/ignore`s. I'm happy either way though 👍

Reply:
yeah. I have that too locally, but can't always force people. These two bits are normcore for most working in text repos.